services/src/main/java/org/apache/druid/server/AsyncQueryForwardingServlet.java

```diff
@@ -286,7 +286,7 @@ protected void service(HttpServletRequest request, HttpServletResponse response)
   private void broadcastQueryCancelRequest(HttpServletRequest request, Server targetServer)
   {
     // send query cancellation to all brokers this query may have gone to
-    // to keep the code simple, the proxy servlet will also send a request to the default targetServer.
+    // keep the code simple, the proxy servlet will also send a request to the default targetServer.
```

Comment:
these comments just seem to be repeating what the javadoc says, maybe just remove them completely?

Reply:
Removed the comment.